### ISSUE TITLE
Training and validation proportion

### ISSUE THREAD

**Issue by Z-Jay-Yang** [qinche106/cb-convlstm-eyetracking#10]
Why is it mentioned in the paper that the training set and validation set are divided into 8:2, but in Github Code,the lists in train_list.txt and val_list.txt are 16:2 (8:1) ?
Thank you
